charts/minecraft/values.yaml

```diff
@@ -180,6 +180,8 @@ minecraftServer:
     existingSecret:
     secretKey: rcon-password
     serviceType: LoadBalancer
+    ## Set the external port if the rcon serviceType is NodePort
+    # nodePort:
```

Comment:
Could actually leave that uncommented so it will be "visible in the schema"

Reply:
I agree with this but I was copying how its handled in the minecraft service. Should I change that too?

Reply:
Good point. Yes, please that one too.

Reply:
I've un-commented both nodePort in values.